src/mono/wasm/templates/templates/browser/browser.0.csproj

```diff
@@ -2,7 +2,6 @@
   <PropertyGroup>
     <TargetFramework>net7.0</TargetFramework>
     <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
-    <WasmMainJSPath>main.js</WasmMainJSPath>
```

Comment:
does this need `<WasmExtraFilesToDeploy Include="index.html" />` or main.js ?

Reply:
They are already in place